diff --git a/cuda_bindings/cuda/bindings/_version.py b/cuda_bindings/cuda/bindings/_version.py
index 03e1a0b..cf68971 100644
--- a/cuda_bindings/cuda/bindings/_version.py
+++ b/cuda_bindings/cuda/bindings/_version.py
@@ -26,9 +26,9 @@ def get_keywords() -> Dict[str, str]:
     # setup.py/versioneer.py will grep for the variable names, so they must
     # each be defined on a line of their own. _version.py will just call
     # get_keywords().
-    git_refnames = "$Format:%d$"
-    git_full = "$Format:%H$"
-    git_date = "$Format:%ci$"
+    git_refnames = " (tag: v12.6.2.post1)"
+    git_full = "92aa73156ec6d0af689f72ca4d8f6bf39871afb9"
+    git_date = "2024-11-12T09:55:45-0800"
     keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
     return keywords
 
