diff --git a/cuda_bindings/cuda/bindings/_version.py b/cuda_bindings/cuda/bindings/_version.py
index 03e1a0b..e85acda 100644
--- a/cuda_bindings/cuda/bindings/_version.py
+++ b/cuda_bindings/cuda/bindings/_version.py
@@ -26,9 +26,9 @@ def get_keywords() -> Dict[str, str]:
     # setup.py/versioneer.py will grep for the variable names, so they must
     # each be defined on a line of their own. _version.py will just call
     # get_keywords().
-    git_refnames = "$Format:%d$"
-    git_full = "$Format:%H$"
-    git_date = "$Format:%ci$"
+    git_refnames = " (tag: v11.8.5.post1)"
+    git_full = "6c8426903e3399d1bd9c5fd65b762eece4af8b5a"
+    git_date = "2024-11-12T10:41:08-0800"
     keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
     return keywords
 
